Quick update -- I tried to download the code, but it is resulting in a new set of compilation errors that seem to be related to the ESP8266Audio code compatibility issue with ESP32 libraries newer than 1.0.6.  I noticed this issue before, but punted on it before I could implement a fix for this by making a note in the README.  Would you be willing to change the PR back to using name() instead? I believe that this would allow everything to but still use the older library.  The full error is below.  (Also I noticed a missing `)` on line 246.)  Thanks again for your help!

>In file included from C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\AudioFileSourceSPIRAMBuffer.h:27,
                 from C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\AudioFileSourceSPIRAMBuffer.cpp:28:
C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\spiram-fast.h: In member function 'void ESP8266SPIRAM::begin(int, int)':
C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\spiram-fast.h:335:26: error: 'SPECIAL' was not declared in this scope
             pinMode(sck, SPECIAL);
                          ^~~~~~~
C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\spiram-fast.h:335:26: note: suggested alternative: 'SERIAL'
             pinMode(sck, SPECIAL);
                          ^~~~~~~
                          SERIAL
In file included from C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\AudioGeneratorMIDI.cpp:65:
C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\libtinysoundfont/tsf.h: In function 'void tsf_channel_midi_control(tsf*, int, int, int)':
C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\libtinysoundfont/tsf.h:2100:1: error: insn does not satisfy its constraints:
 }
 ^
(insn 858 343 344 51 (set (reg:SF 19 f0 [407])
        (mem/u/c:SF (symbol_ref/u:SI ("*.LC248") [flags 0x2]) [0  S4 A32])) "C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\libtinysoundfont/tsf.h":2053 47 {movsf_internal}
     (nil))
during RTL pass: postreload
C:\Users\Alex\Documents\Arduino\libraries\ESP8266Audio\src\libtinysoundfont/tsf.h:2100:1: internal compiler error: in extract_constrain_insn, at recog.c:2210
Please submit a full bug report,
with preprocessed source if appropriate.
See <https://gcc.gnu.org/bugs/> for instructions.

--- a/Code/miniVideoPlayer/miniVideoPlayer.ino
+++ b/Code/miniVideoPlayer/miniVideoPlayer.ino
@@ -188,12 +188,23 @@ void getFilenames(File dir, int fileNo)
         fileCounter = fileCounter+1;
         entry = dir.openNextFile(); // Open next file for reading
       }
+  
+  // arduino-esp32 < 2.0.0 were based on IDF 3.3.5 (and earlier).
+  //               >= 2.0.0 on IDF 4.4 (and later) -> this version includes File::path()
+  #if ESP_IDF_VERSION_MAJOR > 4 || (ESP_IDF_VERSION_MAJOR == 4 && ESP_IDF_VERSION_MINOR >= 4)
       videoFilename = entry.path();
+  #else
+      videoFilename = entry.name();
+  #endif
       Serial.print("Loading video: ");
       Serial.println(videoFilename);
       entry.close();
       entry =  dir.openNextFile();
+  #if ESP_IDF_VERSION_MAJOR > 4 || (ESP_IDF_VERSION_MAJOR == 4 && ESP_IDF_VERSION_MINOR >= 4)
       audioFilename = entry.path();
+  #else
+      audioFilename = entry.name();
+  #endif
       Serial.print("Loading audio: ");
       Serial.println(audioFilename);
       entry.close();

